govcd/media.go

```diff
@@ -19,21 +19,48 @@ import (
 	"github.com/vmware/go-vcloud-director/v2/util"
 )
 
+// Deprecated: use MediaRecord
```

Comment:
This name has been replaced somewhere, but not consistently.
It is still used in non-deprecated functions and in comments.

Please make sure that we use the name `mediaItem` or named composed with it only in deprecated functions and comments. Otherwise, we'll have a confusing situation in our hands.

Reply:
If you find anything else please state, cause I scanned I don't see usage.

Reply:
I am not talking about the type, but the type name. In `media.go` only I can find the following functions that are using a MediaItem.
`UploadMediaImage` `executeUpload`, `RemoveMediaImageIfExists`, `RemoveMediaIfExists`

There are many more in the rest of the code

Reply:
All not deprecated methods now don't have mediaItem anymore